5.5/examples/replica/mysql_replica.json

```diff
@@ -158,11 +158,11 @@
                 "env": [
                   {
                     "name": "MYSQL_MASTER_USER",
```

Comment:
At first I'm not changing the env var, for it would change the "api" of the image... not sure if it is worth changing the parameter name and have it different than the env var.